packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -302,6 +302,15 @@ class DependencyExtractionWebpackPlugin {
 				assetData.type = 'module';
 			}
 
+			if ( compilation.options?.optimization?.runtimeChunk !== false ) {
+				assetData.handle =
+					uniqueName +
```

Comment:
Interesting, the unique name might not be defined based on the failing unit tests. By the way, this change will most likely require updating test snapshots as sometimes the handle is going to be printed.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/64924/commits/dad5d0482424a7db4d547bf0fae0c05096c3f90b.